python_backend/services/rag_service.py

```
@@ -27,28 +28,119 @@ def __init__(self, user_preferences_service=None):
+                logger.info("Simple Text Embedder 생성 및 로드 완료")
+            else:
+                logger.error("Simple Text Embedder 초기화 완전 실패")
+                self.simple_embedder = None
+                
+        except Exception as e:
+            logger.error(f"Simple Text Embedder 초기화 실패: {e}")
+            self.simple_embedder = None
+    
+    def _load_documents(self) -> list:
+        """문서 로드 공통 함수"""
+        from pathlib import Path
```

Ruff (0.12.2)

Redefinition of unused `Path` from line 9

Remove definition: `Path`

(F811)